### PR TITLE
Add site.srht.shrimple.Ouch

### PR DESCRIPTION
**Pull request by theycallhermax**
### Please confirm your submission meets all the criteria

- [x] Please describe the application briefly.
  A clone of Arc Browser for Linux (specifically GNOME)

- [x] The domain used for the application ID is [controlled by the application developers][appid-domain] and the [application id guidelines][appid] are followed.
- [x] I have read the and followed all the [Submission and licence requirements][reqs].
- [x] I have [built][build] and tested the submission locally.
- [x] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about this submission. **Link:**

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

**Comment by theycallhermax** [comment]
bot, build site.srht.shrimple.Ouch

**Comment by theycallhermax** [comment]
bot, build site.srht.shrimple.Ouch

**Comment by theycallhermax** [comment]
bot, build site.srht.shrimple.Ouch

**Comment by theycallhermax** [comment]
bot, build site.srht.shrimple.Ouch

File: site.srht.shrimple.Ouch.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--filesystem=home"
+    ],
+    "build-options": {
+        "append-path": "/usr/lib/sdk/rust-stable/bin",
+        "build-args": [
+            "--share=network"
```
Comment (hfiguiere) [inline site.srht.shrimple.Ouch.json:21]: this is not allowed. build must be offline.

File: flathub.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": [ "x86_64" ]
```
Comment (hfiguiere) [inline flathub.json:2]: this is disappointing.

File: site.srht.shrimple.Ouch.json (hunk 1 above)
Comment (hfiguiere) [reply site.srht.shrimple.Ouch.json:21]: this has been the reason of ALL the build failures.

File: flathub.json (hunk 2 above)
Comment (theycallhermax) [reply flathub.json:2]: i am not really interested in installing another rust target rn

**Comment by theycallhermax** [comment]
bot, build site.srht.shrimple.Ouch

File: flathub.json (hunk 2 above)
Comment (hfiguiere) [reply flathub.json:2]: what about aarch64 users then?

File: site.srht.shrimple.Ouch.json [hunk 3]
```diff
@@ -0,0 +1,57 @@
+{
+    "id": "site.srht.shrimple.Ouch",
```
Comment (bbhtt) [inline site.srht.shrimple.Ouch.json:2]: https://shrimple.srht.site/ says 404 not found, how do we verify that you own the domain?

File: site.srht.shrimple.Ouch.json [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+                "install -Dm 644 ${FLATPAK_ID}.desktop --target-directory ${FLATPAK_DEST}/share/applications",
+                "install -Dm 644 ${FLATPAK_ID}.metainfo.xml --target-directory ${FLATPAK_DEST}/share/metainfo",
+                "install -Dm 644 ${FLATPAK_ID}.svg --target-directory ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/",
+                "install -Dm 644 site.srht.shrimple.svg --target-directory ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+		    "sha256": "0e3983667e20016977a8c02407a86c3470532238f2c70cf934c2696246b72ada",
+                    "url": "https://git.sr.ht/~shrimple/ouch/refs/download/0.3.3/ouch-0.3.3.tar.gz"
```
Comment (bbhtt) [inline site.srht.shrimple.Ouch.json:52]:
```suggestion
                {
                    "type": "archive",
		                "sha256": "0e3983667e20016977a8c02407a86c3470532238f2c70cf934c2696246b72ada",
                    "url": "https://git.sr.ht/~shrimple/ouch/refs/download/0.3.3/ouch-0.3.3.tar.gz"
```

File: site.srht.shrimple.Ouch.json (hunk 3 above)
Comment (bbhtt) [reply site.srht.shrimple.Ouch.json:2]: Ah nvm, it is code hosting.

**Comment by bbhtt** [comment]
bot, build site.srht.shrimple.Ouch

**Pull request closed without merging**